net.waterfox.waterfox.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: net.waterfox.waterfox
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: waterfox
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=wayland"
+  - "--filesystem=xdg-download:rw"
+  - "--share=network"
+  - "--socket=pulseaudio"
+  - "--device=all"
+  - "--talk-name=org.freedesktop.Notifications"
+modules:
+  - name: dbus-glib
+    config-opts:
+      - "--disable-static"
+      - "--disable-gtk-doc"
+    sources:
+      - url: 'https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.112.tar.gz'
+        sha256: 7d550dccdfcd286e33895501829ed971eeb65c614e73aadb4a08aeef719b143a
+        type: archive
+    cleanup:
+      - '*.la'
+      - /bin
+      - /etc
+      - /include
+      - /libexec
+      - /share/gtk-doc
+      - /share/man
+  - name: waterfox
+    buildsystem: simple
+    build-commands:
+      - ./install.sh
```

Comment:
why aren't the build commands here directly?

Reply:
It was for testing, need to fig that now